README.md

```diff
@@ -170,6 +170,15 @@ Uses the same possible selectors as [`RenderContext#find(selector)`](#rendercont
 Returns the name of node.
 This can only be called on a wrapper of a single node.
 
+```jsx
+function MyComponent() {
+  return <Node />;
+}
+
+const context = shallow(<MyComponent />);
+expect(context.name()).toBe('Node');
```

Comment:
can you verify this shouldn't be `context.output().name()` ?  I feel like `context.name() === 'MyComponent'`

Reply:
You're right. Now this is different behaviour than what's implemented in enzyme.
In enzyme the example from README [would work](https://github.com/airbnb/enzyme/blob/master/docs/api/ShallowWrapper/name.md).

I don't really know what to do with that. Do you have any ideas?

Reply:
We aren't really trying to be 100% enzyme compatible, just "inspired by enzyme" 

In many situations I could see you wanting `context.output()` by default, but since this is a different rendering example than say:
```jsx
shallow(<div><MyComponent class="first" /><MyComponent class="second" /></div>);
```
where `context.name()` would be `'div'` either way, `.find('.first').name()` is MyComponent, and `.find('.first').output().name()` would then be "Node" I think it makes sense to show the difference in the example here, just show `expect(context.name()).toBe('MyComponent')` AND `expect(context.output().name()).toBe('Node')`  though `.output()` is returning raw VDOM instead of a FindWrapper... this puts us in an interesting place... we need an `.output()` that returns FindWrappers

Reply:
So what you're saying is that the implementation is good (not the same as enzyme's but this is not an issue for us) and I only have to adjust the documentation?

Reply:
I'm saying we probably need to figure out a solution for `.outputWrapped` or something to make `context.rendered().name()` or something workable for this test... got any thoughts?  I like the idea of `.output() == FindWrappered` and `.rendered() == VNodes` - but it breaks back-compat with 1.0 already, so if we keep `.output() === VNodes` maybe use `.resolved() === FindWrappered` ?

Reply:
I would be in favor of `.instance() == VNode` and `.output() == FindWrapper` but it also breaks backwards compatibility.

Or without breaking back compat make `.instance()` return `VNode`, leave `.output()` as it is (doing effectively the same thing as `.instance()`) and then make `.instanceWrapper()` return `FindWrapper`.

Reply:
I'm gonna bring this conversation to #66 -- need @mzgoddard to chime in on it too I think.

Reply:
I'm game for `name` returning `this[0].nodeName` or `'MyComponent'`.

I think a name to see the next level in for a component node could be `inspect` or `stepIn`. I think `instance` would be confusing along side `component`.

In that case `.stepIn().name()` would give you `'Node'` and `.resolved().name()` would give `'span'` or whatever raw dom node name Node returns. `.stepIn().stepIn().name()` would also give `'span'` if Node was the last component type in the chain of nested nodes.

Reply:
This readme is still "incorrect" - could you update this / show an example of the difference between enzyme?